src/ensembl/src/shared/components/panel/Panel.test.tsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import faker from 'faker';
+
+import Panel, { PanelProps } from './Panel';
+
+const panelContent = <p>{faker.lorem.words()}</p>;
+
+const panelHeader = <p>{faker.lorem.words()}</p>;
+
+const onClose = jest.fn();
+
+const panelClassNames = {
+  panel: faker.lorem.word(),
+  header: faker.lorem.word(),
+  body: faker.lorem.word(),
+  closeButton: faker.lorem.word()
+};
+
+const defaultProps: PanelProps = {
+  header: panelHeader,
+  children: panelContent,
+  classNames: panelClassNames
+};
+
+const renderPanel = (props: PanelProps) => {
+  return <Panel {...props}></Panel>;
+};
+
+describe('<Tabs />', () => {
+  let wrapper: any;
+
+  beforeEach(() => {
+    wrapper = mount(renderPanel({ ...defaultProps }));
+  });
```

Comment:
We shouldn't be running this for every test case, as we are remounting the component with different props in some test cases. How about using a render function that would take optional props as arguments to mount instead?

Reply:
That's done